src/ansys/fluent/core/utils/search.py

```diff
@@ -190,13 +190,12 @@ def _search(
     api_objects = []
     api_tui_objects = []
     api_object_names = set()
+    results = set()
```

Comment:
Maybe we use a list to maintain the same order from the api_tree file (which follows the settings API order)

Reply:
Ok. We will update this.

Reply:
Done. Thank you.